packages/client/tests/functional/issues/21552-datetime-representation-compatibility/tests.ts

```diff
@@ -51,5 +51,9 @@ testMatrix.setupTestSuite(
         which are not supported with data proxy.
       `,
     },
+    skipEngine: {
+      from: ['wasm'],
+      reason: 'Fails with PrismaClientValidationError: Invalid client engine type, please use `library` or `binary`',
```

Comment:
That's a weird one

Reply:
It comes from: https://github.com/prisma/prisma/blob/021d5ed22acda0c6e90b9cb3dc5e8e0b5e2d0d04/packages/client/src/runtime/core/init/getEngineInstance.ts#L50-L53